guides/source/configuring.md

```diff
@@ -668,6 +668,7 @@ Defaults to `'signed cookie'`.
 
 * `config.action_view.default_enforce_utf8` determines whether forms are generated with a hidden tag that forces older versions of Internet Explorer to submit forms encoded in UTF-8. This defaults to `false`.
 
+* `config.action_view.annotate_rendered_view_with_filenames` determines whether to annotate rendered view with template file names. This defaults to `false`.
```

Comment:
```suggestion
* `config.action_view.annotate_rendered_view_with_filenames` determines whether to annotate rendered views with template file names. This defaults to `false`.
```

Reply:
@joelhawksley I don't think so as `rendered view` is only one - it is combination of different files/templates. But once rendered it is one.